### PR TITLE
Adding amlrsdk team alias as maintainer, and renaming package to azuremlsdk

### PR DESCRIPTION
**Pull request by heemanshusuri**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -4,9 +4,10 @@ Title: R interface to AzureML SDK
 Version: 0.4.0
 Authors@R: c(
     person("Himanshu", "Chandola", email = "Himanshu.Chandola@microsoft.com", role = c("aut")),
-    person("Billy", "Hu", email = "ninhu@microsoft.com", role = c("aut", "cre")),
+    person("Billy", "Hu", email = "ninhu@microsoft.com", role = c("aut")),
     person("Heemanshu", "Suri", email = "Heemanshu.Suri@microsoft.com", role = c("aut")),
     person("Diondra", "Peck", email = "Diondra.Peck@microsoft.com", role = c("aut")),
+    person("Azure ML R SDK", email = "amlrsdk@microsoft.com", role = c("cre")),
```
Comment (ninghu) [inline DESCRIPTION]: AzureML R SDK Team

Comment (ninghu) [reply DESCRIPTION]: I think it's fine to have "AzureML" as it's just warning from R hub

File: vignettes/installation.Rmd [hunk 2]
```diff
@@ -24,7 +24,7 @@ remotes::install_github('https://github.com/Azure/azureml-sdk-for-r')
 ```
 
 ## 3. Install AzureML Python SDK
-Lastly, use the `azureml` R library to install the compiled code. By default, `azureml::install_azureml` will install the [latest version of the AzureML Python SDK](https://pypi.org/project/azureml-sdk/) in a conda environment called `r-azureml`.
+Lastly, use the `azureml` R library to install the compiled code. By default, `azureml::install_azureml()` will install the [latest version of the AzureML Python SDK](https://pypi.org/project/azureml-sdk/) in a conda environment called `r-azureml`.
```
Comment (ninghu) [inline vignettes/installation.Rmd]: azuremlsdk::install_azureml()

Comment (heemanshusuri) [reply vignettes/installation.Rmd]: changed

**Pull request merged**